apps/web/next.config.js

```diff
@@ -333,44 +334,46 @@ const nextConfig = {
           },
         ],
       },
-      ...[
```

Comment:
Add the logging headers only when the redirect is actually happening. [Same check is already there for redirects](https://github.com/calcom/cal.com/blob/81116abd48072996d8f162e73267617f7adb15ae/apps/web/next.config.js#L230).